app/config/config.yml

```diff
@@ -81,11 +84,12 @@ assetic:
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/tables.less"
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/messages.less"
             output: css/min.css
-
         common_css:
             inputs:
-              - "@WebsiteBundle/Resources/public/css/common.css"
+                - "bundles/website/css/common.css"
```

Comment:
Looks like you are using `css` from `webroot`. Strange. May I check this configuration later today, if you don't mind?